package.json

```diff
@@ -82,6 +82,7 @@
   },
   "dependencies": {
     "global": "^4.3.2",
+    "natives": "^1.1.6",
```

Comment:
Is this used anywhere?

Reply:
I think I had some build failure at some point. Don't remember what exactly. I'll revert it.

Reply:
Reverted during rebase.

Reply:
Yeah, I think https://travis-ci.org/videojs/videojs-vr/builds/575075840 is the build failure.
```
21 08 2019 21:50:35.760:ERROR [config]: Invalid config file!
  ReferenceError: internalBinding is not defined
    at internal/util/inspect.js:31:15
    at req_ (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:140:5)
    at require (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:113:12)
    at util.js:25:21
    at req_ (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:140:5)
    at require (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:113:12)
    at fs.js:42:21
    at req_ (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:140:5)
    at Object.req [as require] (/home/travis/build/videojs/videojs-vr/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/home/travis/build/videojs/videojs-vr/node_modules/unzip/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
```